api-reference/10 UI Components/dxButton/1 Configuration/useSubmitBehavior.md

```diff
@@ -12,6 +12,8 @@ If this property is set to **true**, users can click the button to validate the
 
 If the [onClick](/api-reference/10%20UI%20Components/dxButton/1%20Configuration/onClick.md '/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick') event handler is specified, it is executed before validation and form submission.
 
+You cannot call the [preventDefault()](/Documentation/ApiReference/Common/Object_Structures/EventObject/Methods/#preventDefault) method inside the **onClick** function. Handle a form's [submit](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) event to prevent the form submission.
```

Comment:
```suggestion
You cannot call the [preventDefault()](/Documentation/ApiReference/Common/Object_Structures/EventObject/Methods/#preventDefault) method inside the **onClick** function. Handle a form's [submit](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) event to prevent form submission.
```